### ISSUE TITLE
Broken repeater field population in v2.1.3

### ISSUE THREAD

**Issue by jessedobbelaere** [verbb/formie#1716]
### Describe the bug

I use twig to populate a Repeater field as described here: https://verbb.io/craft-plugins/formie/docs/template-guides/populating-forms#repeater
I noticed nothing gets populated after upgrading to `v2.1.3`. 

When I debug the `Rendering.php` class of Formie in the `populateFormValues` method, I noticed an error gets caught and silenced in the catch statement (https://github.com/verbb/formie/blob/2.1.3/src/services/Rendering.php#L508-L510)

```
Array to string conversion
/var/www/html/vendor/verbb/formie/src/services/Rendering.php:491
```

The error gets thrown by this line that got introduced in v2.1.3 to sanitize the input: https://github.com/verbb/formie/blob/2.1.3/src/services/Rendering.php#L490-L491. 

It's doing a string operation, but a repeater field needs an array... These are the local variables on that line, `$value` is not a string:

```php
$key = 'shoppingCartField'
$value = array (
  0 => array (
    'productField' => array (
      0 => 5903,
    ),
    'quantityField' => '1',
  ),
)
```

👉 Downgrading from `v2.1.3` to `v2.1.2` fixes the issue ✅ 

### Steps to reproduce

1. Create a form with a repeater field with an entries field inside (could be anything)
2. Populate the repeater field from Twig as described on https://verbb.io/craft-plugins/formie/docs/template-guides/populating-forms#repeater
3. Nothing gets populated. An error `Array to string conversion` is silently caught in a catch statement.

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.7.1

### Plugin version

2.1.3

### Multi-site?

Yes

**Comment by ishetnogferre** [comment]
Aha, was just going the same debug path 😅  
+1

**Comment by engram-design** [comment]
Good catch, this was meant to sanitize values just to protect the (somewhat) rare instance that developers weren't escaping values provided to `populateFormValues()`.

Should be fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.3"`.

**Comment by engram-design** [comment]
Fixed in [2.1.4](https://github.com/verbb/formie/releases/tag/2.1.4)
